pyproject.toml

```diff
@@ -55,7 +55,7 @@ line-length = 120
 target-version = "py311"
 
 [tool.ruff.lint]
-select = ["I", "E", "W", "D", "PL"]
+select = ["I", "E", "W", "D", "PL", "F401"]
```

Comment:
I wonder if it might be better to use `F` for all the rules in pyflake. 

https://docs.astral.sh/ruff/rules/#pyflakes-f

Reply:
That would definitely make it simpler. What do you think, @jyotiraj-code ?